data/mods/My_Sweet_Cataclysm/sweet_recipe.json

```diff
@@ -13,5 +13,135 @@
     "qualities": [ { "id": "COOK", "level": 2 } ],
     "tools": [ [ [ "surface_heat", 2, "LIST" ] ] ],
     "components": [ [ [ "sugar", 10 ] ], [ [ "water", 1 ], [ "water_clean", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
```

Comment:
All those recipe can move to vanilla.

Reply:
moved most of them, left the last two here because people don't really preserve chocolate milk like that IRL as far as I can tell